app/src/main/java/org/jellyfin/androidtv/preference/UserPreferences.kt

```diff
@@ -87,6 +88,11 @@ class UserPreferences(context: Context) : SharedPreferenceStore(
 		 */
 		var cinemaModeEnabled = booleanPreference("pref_enable_cinema_mode", true)
 
+		/**
+		 * Enable still watching
+		 */
+		var stillWatchingBehavior = enumPreference("pref_enable_still_watching", StillWatchingBehavior.DEFAULT)
```

Comment:
The `pref_` prefix is only used for old preference so we don't have to migrate their keys. Remove it for new ones.